packages/mobile/src/home/SendOrRequestBar.tsx

```diff
@@ -11,11 +11,13 @@ import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
 import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
 import useSelector from 'src/redux/useSelector'
-import { tokensWithTokenBalanceSelector } from 'src/tokens/selectors'
+import { tokensListSelector, tokensWithTokenBalanceSelector } from 'src/tokens/selectors'
 
 export default function SendOrRequestBar() {
   const sendableTokens = useSelector(tokensWithTokenBalanceSelector)
-  const buttonsDisabled = sendableTokens.length === 0
+  const sendButtonsDisabled = sendableTokens.length === 0
+  const requestButtonDisabled = useSelector(tokensListSelector).length === 0
+  const qrScanDisabled = false // TODO: Move logic to when actually scanning
```

Comment:
I think the QR button should never be disabled from here, so we can just remove this line and the disabled prop for the component?